content/en/docs/userguide/metric.md

```diff
@@ -0,0 +1,148 @@
+---
+draft: false
+linktitle: Querying L4 Metrics from Prometheus
+menu:
+  docs:
+    parent: user guide
+    weight: 4
+title: Querying L4 Metrics from Prometheus
+toc: true
+type: docs
+
+---
+
+### Preparation
+
+1. Install Kmesh:
+
+Please refer [quickstart](https://kmesh.net/en/docs/setup/quickstart/)
+
+Note that the following should be added to the Kmesh annotation:
+
+```console
+annotations:
+    prometheus.io/path: "status/metric"
+    prometheus.io/port: "15020"
+    prometheus.io/scrape: "true"
+```
+
+2. Using Kmesh to manage `default` namespace
+
+```console
+kubectl label namespace default istio.io/dataplane-mode=Kmesh
+
+namespace/default labeled
+```
+
+3. Install the Prometheus Addon:
+
+Istio provides a basic sample installation to quickly get Prometheus up and running:
+
+```console
+kubectl apply -f https://raw.githubusercontent.com/kmesh-net/kmesh/main/samples/addons/prometheus.yaml
+```
+
+4. Deploy the tcp-echo and sleep application
+
+```console
+kubectl apply -f https://raw.githubusercontent.com/kmesh-net/kmesh/main/samples/tcp-echo/tcp-echo.yaml 
+
+kubectl apply -f https://raw.githubusercontent.com/kmesh-net/kmesh/main/samples/sleep/sleep.yaml
+
+kubectl get po -A 
+NAMESPACE            NAME                                          READY   STATUS    RESTARTS   AGE
+default              sleep-bc9998558-pbfvk                         1/1     Running   0          7m
+default              tcp-echo-7f676db574-mzmql                     1/1     Running   0          7m
+```
+
+**Note:** Confirm that sleep and tcp-echo are indeed [managed by kmesh](https://kmesh.net/en/docs/setup/quickstart/#deploy-the-sample-applications).
+
+### Querying Metrics from Prometheus
+
+Metrics monitored by Kmesh L4 at this stage:
+
+**Workload:**
+
+|Name|Describe|
+|---|---|
+|kmesh_tcp_workload_connections_opened_total|The total number of TCP connections opened to a workload|
+|kmesh_tcp_workload_connections_closed_total|The total number of TCP connections closed to a workload|
+|kmesh_tcp_workload_received_bytes_total|The size of the total number of bytes received in response to a workload over a TCP connection|
+|kmesh_tcp_workload_sent_bytes_total|The size of the total number of bytes sent in response to a workload over a TCP connection|
+|kmesh_tcp_workload_conntections_failed_total|The total number of TCP connections failed to a workload|
+
+**Service:**
+
+|Name|Describe|
+|---|---|
+|kmesh_tcp_connections_opened_total|The total number of TCP connections opened to a service|
```

Comment:
is this still valid, vagulaly remember is it with _service_

Reply:
metric name changed.

https://github.com/kmesh-net/kmesh/blob/155a8a5dcf542663b7108129dace7729677370d8/pkg/controller/telemetry/utils.go#L148-L177